client/src/App.js

```diff
@@ -0,0 +1,29 @@
+import { Route, Routes } from 'react-router-dom';
+import './component/Navbar';
+import './App.css'; import React from 'react';
+import Navbar from './component/Navbar';
+import Signup from './component/Signup';
+import Home from './component/Home';
+import Register from './component/Register';
+import Detail from './component/Detail';
+import PlayerRegister from './component/PlayerRegister';
+console.log("hit")
```

Comment:
Comment out or remove the console.log before pushing any code.

Reply:
Done